src/api/data_types/chunking/upload/capability.rs

```diff
@@ -53,6 +56,7 @@ impl<'de> Deserialize<'de> for ChunkUploadCapability {
             "bcsymbolmaps" => ChunkUploadCapability::BcSymbolmap,
             "il2cpp" => ChunkUploadCapability::Il2Cpp,
             "preprod_artifacts" => ChunkUploadCapability::PreprodArtifacts,
+            "proguard" => ChunkUploadCapability::Proguard,
```

Comment:
The server already returns this since https://github.com/getsentry/sentry/commit/e0b7ff2fb72e49bfe862c6b90e42257990ea38e1